src/main/com/example009/radek/v3/Calculator.java

```diff
@@ -0,0 +1,66 @@
+package main.com.example009.radek.v3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Calculator {
+
+    private double result;
+    private List<Double> list = new ArrayList<>();
+
+    Calculator(double defaultResult) {
+
+        this.result = defaultResult;
+    }
+
+    void plus(double value) {
+
+        this.result = result + value;
+    }
+
+    void minus(double value) {
+
+        this.result = result - value;
+    }
+
+    void multiply(double value) {
+
+        this.result = result * value;
+    }
+
+    void divide(double value) {
+
+        if (value == 0) {
+            System.out.println("You can't divide by zero!");
+            System.exit(0);
+        }
+        this.result = result / value;
+    }
+
+    double getResult() {
+
+        return this.result;
+    }
+
+    void setResult(double input) {
+
+        this.result = input;
+    }
+
+    void getStoredResults() {
+
+        FileService fileService = new FileService();
```

Comment:
Pekne si oddelil praci se soubory, ale vytvareni te FileService je v kazde metode je spatne ... to je ukazatel do pameti a pri kazdem vytvoreni vytvaris novou servicu nove pametove prostredky 
vytvor si tu servicu jako privatni promennou a pri vytvoreni teto classy ji nainicializuj 
pak jen v konkretnich metodach kde ji potrebujes budes volat fileService.xxx tzn volas tu stejnou servicu jendou vytvorenou 
pamatuj ze vzdy kdyz pouzijes new, tak vytvaris blok v pameti a je treba na to myslet

Reply:
Zkoušel jsem to i se scannerem a ten mi potom dělal chybu, protože stejné slovo zůstávalo v historii a pokaždé bych musel volat reset, ale FileService jsem přesunul